store/sqlstore/post_store.go

```diff
@@ -1238,6 +1239,18 @@ func (s *SqlPostStore) search(teamId string, userId string, params *model.Search
 		searchClause := fmt.Sprintf("AND to_tsvector('english', %s) @@  to_tsquery('english', :Terms)", searchType)
 		searchQuery = strings.Replace(searchQuery, "SEARCH_CLAUSE", searchClause, 1)
 	} else if s.DriverName() == model.DATABASE_DRIVER_MYSQL {
+		if searchType == "Message" {
+			var err error
+			terms, err = removeMysqlStopWordsFromTerms(terms)
```

Comment:
```suggestion
			terms, err := removeMysqlStopWordsFromTerms(terms)
```

Reply:
This case we are shadowing the var `terms`, so we need to re-use the same var :)